src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -6552,7 +6522,7 @@ static NullableFlowState getConversionResultState(TypeWithState operandType)
                 conversionOpt,
                 conversionOperand,
                 conversion.UserDefinedFromConversion,
-                TypeWithAnnotations.Create(conversion.BestUserDefinedConversionAnalysis.FromType),
+                TypeWithAnnotations.Create(conversion.BestUserDefinedConversionAnalysis!.FromType),
```

Comment:
Do we know this as a result of the above `conversion.IsValid` check? If so, let's tag for `MemberNotNullWhen` when that becomes available.

Reply:
I don't think `[MemberNotNull]` will handle this case because the set of fields on `Conversion` that are not `null` will depend on the conversion kind.

---
In reply to: [463277703](https://github.com/dotnet/roslyn/pull/45640#discussion_r463277703) [](ancestors = 463277703)